ports/esp32/boards/SBD_BEES3/board.json

```diff
@@ -0,0 +1,11 @@
+{
+  "deploy": ["../deploy.md"],
+  "docs": "",
+  "features": ["BLE", "WiFi", "USB-C", "RGB LED"],
+  "images": ["bees3.jpg"],
+  "mcu": "esp32s3",
+  "product": "Bee S3",
+  "thumbnail": "",
+  "url": "",
```

Comment:
Do you have a URL to link back to where details of the board can be found? Maybe the Tindie page?